### ISSUE TITLE
Export entries to Excel/Numbers File fails with 500 Internal Server Error

### ISSUE THREAD

**Issue by miklund** [umbraco/Umbraco.Forms.Issues#761]
### Bug summary

We have encountered an issue for exporting entries to Excel with the built in export functionality. When trying to export, it will display the loading indicator, but it will fail silently in the background with 500 Internal Server Error.

### Specifics

**URL in Umbraco**
http://localhost:3000/umbraco#/forms/Form/entries/43b8894c-7172-48ad-8178-224130344603?mculture=sv

**Network call that fails in debug console**
http://localhost:3000/umbraco/backoffice/UmbracoForms/Export/PostGenerateExport 

**Version**
Found in Umbraco Forms 9.2.2, Umbraco CMS 9.3.1
Upgraded to Umbraco Forms 9.3.0 but the problem persists

**Browser**
Tried Firefox 99.0 and Safari 15.3. Client that reported the problem is running Chrome.

**Environments**
Found in production and reproducible in our local dev environment

We're running dotnet 5.0.407 on Linux, both in dev and production. In production we're running this particular image
mcr.microsoft.com/dotnet/aspnet:5.0-buster-slim. No extra packages or configurations.

### Steps to reproduce

1. Login to Umbraco as admin
2. Go to Forms section
3. Expand any form and click "Entries"
4. Click Export link
5. Click "Choose a format to export form records to: Excel/Numbers File"

### Expected result

A file download to start with the Excel spreadsheet for the exported entries.

### Actual result

Loading indicator that never ends. In the background it will fail with 500 Internal Server Error

```
POST http://localhost:3000/umbraco/backoffice/UmbracoForms/Export/PostGenerateExport

Possibly unhandled rejection: {"data":{"ExceptionMessage":"The type initializer for 'Gdip' threw an exception.","ExceptionType":"System.TypeInitializationException, System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e","StackTrace":"   at System.Drawing.SafeNativeMethods.Gdip.GdipGetGenericFontFamilySansSerif(IntPtr& fontfamily)\n   at System.Drawing.FontFamily.GetGdipGenericSansSerif()\n   at System.Drawing.FontFamily.get_GenericSansSerif()\n   at System.Drawing.Font.CreateFont(String familyName, Single emSize, FontStyle style, GraphicsUnit unit, Byte charSet, Boolean isVertical)\n   at System.Drawing.Font..ctor(String familyName, Single emSize, FontStyle style, GraphicsUnit unit, Byte gdiCharSet, Boolean gdiVerticalFont)\n   at OfficeOpenXml.ExcelRangeBase.AutoFitColumns(Double MinimumWidth, Double MaximumWidth)\n   at Umbraco.Forms.Core.Providers.Export.ExportToExcel.ExportToFile(RecordExportFilter filter, String filepath)\n   at Umbraco.Forms.Web.Editors.ExportController.PostGenerateExport(RecordExportFilter model)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\n--- End of stack trace from previous location ---\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextExceptionFilterAsync>g__Awaited|25_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)"},"status":500,"config":{"method":"POST","transformRequest":[null],"transformResponse":[null],"jsonpCallbackParam":"callback","url":"backoffice/UmbracoForms/Export/PostGenerateExport","data":{"startIndex":1,"length":20,"form":"43b8894c-7172-48ad-8178-224130344603","sortBy":"created","sortOrder":"descending","states":["Approved","Submitted"],"localTimeOffset":-120,"exportType":"94ed105a-87b3-4e1f-97cb-9a320aee2745"},"headers":{"Accept":"application/json, text/plain, */*","Content-Type":"application/json;charset=utf-8","X-Requested-With":"XMLHttpRequest","X-UMB-XSRF-TOKEN":"CfDJ8LkH30bt-jpCgYburwNavNFaX5G9v5h4zNHfYZD8twwX89TLEBzFgiEMDet8KVeZ8a0TyRkrPpmXLCCIEO7LggnuINImbiThb5tNWVy3rde2c-tknMYj-e_oWwsqmXDJaeBDDc_6oun2NmOQDSuvSNcbyCUo28SV6n_xOAy0ea0vm1Jh_fbRLM8xdtjHNOcegg"}},"statusText":"Internal Server Error","xhrStatus":"complete"}
```

## Screenshot

Client sensitive information has been redacted.

<img width="1918" alt="Screenshot 2022-04-13 at 16 33 40" src="https://user-images.githubusercontent.com/1799849/163204701-9e271ada-96fc-4bc5-9df3-19e19c4c7e2f.png">

**Comment by AndyButland** [comment]
Thanks for letting us know about this.  I've had a look into it, first thinking this was going to be quite problematic to get to to work on Linux due to that call you've found into `System.Drawing`, which has [some limitations](https://docs.microsoft.com/en-us/dotnet/api/system.drawing?view=net-6.0#remarks) on non-Windows, and is used by an Excel library Forms uses for this feature.

Fortunately though it's only one non-essential feature - auto-fitting the columns - that this is used for, so it looks like this can be fixed omitting these calls unless running on Windows.  I'll add this fix to the upcoming 8.12 release.

**Comment by miklund** [comment]
Thank you @AndyButland for your quick response. I'm looking forward to the 9.4.0 release. :)
